src/ui/widget-forms/HandpickedContentsConfigFormContainer.js

```diff
@@ -22,7 +23,9 @@ export const mapStateToProps = (state, ownProps) => {
   if (multipleContentsMode) {
     widgetConfig = propWidgetConfig;
   } else {
-    delete propWidgetConfig.contents;
+    if (propWidgetConfig != null) {
```

Comment:
should this be loosely different that null? what about `undefined` propWidgetConfig?

can we avoid using `delete` in favor of destructuring?

Reply:
I checked with the trigger function and there `propWidgetConfig` is either an object or null. Also, `!=` operator comparing to `null` also checks with `undefined`, so both cases are guaranteed. 
As for `delete`, I need an object property `contents` to be removed, I think I am not aware of how to do it otherwise